src/components/translation-builder/edit-translation.modal.tsx

```diff
@@ -25,6 +25,7 @@ const EditTranslationModal: React.FC<EditTranslationModalProps> = ({ onClose, or
           labelText={t('translationValue', 'Translated Value')}
           value={newValue}
           onChange={(e) => setNewValue(e.target.value)}
+          data-testid="translation-value-input"
```

Comment:
question: Is the data test id necessary? Can't we target the button from the label text?